biz.aQute.tester.test/base.bndrun

```diff
@@ -1,12 +1,12 @@
 
 -runbundles: \
-	org.junit.platform.launcher;version='[1.0.0,1.0.1)',\
-	org.junit.platform.commons;version='[1.0.0,1.0.1)',\
-	org.junit.platform.engine;version='[1.0.0,1.0.1)',\
-	org.junit.jupiter.api;version='[5.0.0,5.0.1)',\
-	org.junit.jupiter.params;version='[5.0.0,5.0.1)',\
+	org.junit.platform.launcher;version=1.3.1,\
```

Comment:
I suspect these open ended ranges may be cause of the test execution failures in this project. I am surprised these test cases passed on your system when you ran the gradle build. In the gradle build output, you can see the warnings that also should be addressed in this PR.

Reply:
Not directly, but close too it. There had been a few things around the bndrun files for the tests, that came with the new Version but it should work now.